includes/fields/class-color.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Color field.
+ *
+ * @package HivePress\Fields
+ */
+
+namespace HivePress\Fields;
+
+use HivePress\Helpers as hp;
+
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Color.
+ */
+class Color extends Text {
```

Comment:
Is it worth it to make it based on Text? Disabling all the inherited parameters may be an issue.